pkg/testing/cli/harness.go

```diff
@@ -76,6 +76,14 @@ func CreateExitHandlerFunc(t *testing.T, tee *Tee, expectedExitCode int, expecte
 	return exitHandler
 }
 
+func PreventExitHandlerFunc(t *testing.T) func(int) bool {
```

Comment:
Does this mean it will return a function which have return type as `bool` and take a `int` parameter? Just want to understand.